Cli-Shared/Program.cs

```diff
@@ -747,6 +778,76 @@ private static void EnableTraceLogging(OperationArguments operationArguments, st
             }
         }
 
+        private static bool BitbucketCredentialPrompt(string titleMessage, TargetUri targetUri, out string username, out string password)
+        {
+            Credential credential;
+            if ((credential = BasicCredentialPrompt(targetUri, titleMessage)) != null)
+            {
+                username = credential.Username;
+                password = credential.Password;
+
+                return true;
+            }
+
+            username = null;
+            password = null;
+
+            return false;
+        }
+
+        private static bool BitbucketOAuthPrompt(string title, TargetUri targetUri, Bitbucket.AuthenticationResultType resultType,
+            string username)
```

Comment:
nit: please either keep all parameters on the same line, or left-align them vertically immediately after the `(` character.